addon/components/detail-instructors.js

```diff
@@ -47,15 +47,16 @@ export default class DetailInstructorsComponent extends Component {
 
   get instructorCount() {
     if (!this.args.ilmSession) {
-      return null;
+      return 0;
     }
-    return this.args.ilmSession.hasMany('instructors').ids().length;
```

Comment:
Ugh. I wish we could get to Ember Data v4 and not have to do this!

Reply:
:crossed_fingers: